ipa-uk.py

```diff
@@ -153,7 +155,7 @@ def ipa(text: str, check_accent: bool) -> str:
     # including stress mark for single-syllable words if check_accent is set to true
     number_of_vowels = len(re.findall(r"[ɑɛiɪuɔ]", phonetic))
     if number_of_vowels == 1 and check_accent:
-        phonetic = re.sub(r"([ɑɛiɪuɔ])", r"ˈ\1", phonetic)
+        phonetic = re.sub(r"([ɑɛiɪuo])", r"ˈ\1", phonetic)
```

Comment:
Two lines above we are counting vowels and using this list of characters: ɑɛiɪuɔ
And here we are using o instead of ɔ. As far as I can understand from the code, there is no chance, that the word at this point of the processing will have any o's, as they are unconditionally replaced with ɔ.

Reply:
We need to rewrite this. We need to add 'o' to the 'number_of_vowels' because otherwise stressing of 'o's work pretty randomly.